compiler/src/main/kotlin/main.kt

```diff
@@ -40,6 +42,7 @@ fun compile(path: Path) {
         TypeChecker(symbolTable).visit(ast)
         ErrorLogger.assertNoErrors()
 
+        File("ast.gs").outputStream().use { out -> ASTGrapher(out).visit(ast) }
```

Comment:
This should be an argument. Like #81, we should use a proper way of parsing arguments.

Reply:
Let's move that change to a separate PR for commandline arguments

Reply:
Yes, that's fine. Maybe we should open an issue on it.

Reply:
I have create a issue, see #86